src/helpers/enrich-dataset.ts

```diff
@@ -223,4 +223,14 @@ function objectWithoutKeys(obj, keys): Record<string, any> {
         if (keys.indexOf(key) === -1) newObject[key] = obj[key];
         return newObject;
     }, {});
-}
\ No newline at end of file
+}
+
+function timestampToIsoDate (val: number): string {
+    if (_.isNil(val)) return undefined;
+
+    const date = new Date(val);
+    if (date instanceof Date && !isNaN(date.valueOf())) {
+        return new Date(val).toISOString();
```

Comment:
Can't you just use the existing date object? `return date.toIsoString()`

Reply:
Good point! I have used `return date.toIsoString()` instead.